packages/core/src/types/config.ts

```diff
@@ -16,17 +16,20 @@ export enum VerificationLevel {
 	Device = 'device',
 }
 
-export type IDKitConfig = {
+export type BridgeConfig = {
+	/** URL to a third-party bridge to use when connecting to the World App. Optional. */
+	bridge_url?: string
 	/** Unique identifier for the app verifying the action. This should be the app ID obtained from the Developer Portal. */
 	app_id: `app_${string}`
```

Comment:
why do we need to include app_id inside bridge config. If this is for allowing generic payloads in the bridge?